src/nvim/mbyte.c

```diff
@@ -458,6 +458,19 @@ static bool intable(const struct interval *table, size_t n_items, int c)
   return false;
 }
 
+#ifdef NVIM_VTERM_HAS_USER_FUNC
+///
+/// utf_char2cells() with different argument type for libvterm.
```

Comment:
this sounds a little bit too much of a lie (as it surely does more than call `utf_char2cells`)